docs/hardware/hardware-selection.md

```diff
@@ -0,0 +1,278 @@
+
+    For new users unless you plan on flying hundreds of kilometers or in a high noise environment we reccomend 2.4GHz hardware:
+
+      * Axisflying Thor
+      * RadioMaster Ranger
+      * HappyModel ES24TX Pro
+      * Namimno Flash
+
+    If you want to push the limits of range, a 900MHz system might fit within your needs. Some off the shelf 900MHz transmitters include:
+
+      * Namimno Voyager
+      * Happymodel ES900TX
+      * BetaFPV Micro 915/868MHz
+
+??? info "Size"
+
+    Most ExpressLRS Transmitter modules fit into one of two classes - Micro and Nano. There are some exceptions that fit into multiple or have interesting features that may make them work better for your needs.
+    
+    Micro modules will fit into the JR bay of a standard transmitter such as a TX16s or QX7, with some examples including:
+
+      * RadioMaster Ranger Micro
+      * Namimno Flash
+      * HappyModel ES24TX
+    
+    Nano modules will fit into a lite module bay such as that on a Zorro, T-Pro or X-Lite with some examples including:
+
+      * HappyModel ES24TX Slim Pro
+      * Radimoaster Ranger Nano
+      * Jumper AION Nano
+      * BetaFPV Nano
+
+    Some notable exceptions that can fit into multiple or very specific module bays include:
+
+      * HappyModel ES24TX Lite - Fits onto Jumper T-Lite
+      * Axisflying Thor - Has a wire system to be able to plug into any radio with a CRSF output
+      * Radiomaster Ranger - includes Micro & Nano mounts and wiring for any CRSF compatible radio
+
+??? info "Power"
+
+    Most ELRS Transmitters have a 250mW or 500mW power cap, but for that extra percentage of range, a select few 1W modules exist, packing that punch needed to go even further. These transmitters include large heatsinks, fans and sometimes temperature sensing to cool the power output. Some of these 1W transmitters are:
+
+      * HappyModel ES24TX Pro
+      * RadioMaster Ranger
+      * Axisflying Thor
+      * BetaFPV Micro TX 1W
+      * RadioMaster Boxer
+      * Jumper T-Pro Internal
+      * NamimnoRC Flash OLED
+
+??? info "Screens"
+
+    Some transmitters include a small screen that displays useful information and pairs with a small joystick to quickly change settings on the go. This can be useful when paired with handsets running firmware that does not support lua scripts. The two main types of screens are TFT and OLED:
+
+    <figure>
+      <img class="center-img" src="../../assets/images/hw-selection/screens.png"/>
+    </figure>
+
+    The only transmitter with a TFT screen on the market at the moment is the Axisflying Thor TX.
+
+    Some transmitters with an OLED include:
+
+      * Namimno Flash OLED
+      * RadioMaster Ranger
+      * BetaFPV Micro TX
```

Comment:
might want to include the Jumper Aion Nano module here.. Don't want anyone to feel left out.. :D